packages/dbos-cloud/databases/databases.ts

```diff
@@ -9,11 +9,26 @@ export interface UserDBInstance {
   readonly DatabaseUsername: string;
 }
 
+function isValidPassword(password: string): boolean {
+  if (password.length < 8 || password.length > 128) {
+    return false;
+  }
+  if (password.includes('/') || password.includes('"') || password.includes('@') || password.includes(' ') || password.includes('\'')) {
+    return false;
+  }
+  return true
+}
+
 export async function createUserDb(host: string, dbName: string, appDBUsername: string, appDBPassword: string, sync: boolean) {
   const logger = getLogger();
   const userCredentials = await getCloudCredentials();
   const bearerToken = "Bearer " + userCredentials.token;
 
+  if (!isValidPassword(appDBPassword)) {
+    logger.error("Invalid database password. Passwords must be between 8 and 128 characters long and can contain any ASCII character except @, /, \\, \", ', and spaces")
```

Comment:
Maybe a better place to put this error message is directly inside `isValidPassword` function. Then we can separately log the password length error vs the special character error.

Reply:
Or create a new error type